### ISSUE TITLE
Change CMakeLists.txt to force Python 3.8

### ISSUE THREAD

**Issue by rmchurch** [ukaea/PROCESS#2861]
Current install instructions calls for Python 3.8 (and only Python 3.8), but the cmake call wasn't picking up my conda env python=3.8 (instead would pick up system python 3.9, and fail in build). Changing to add `EXACT` to this line in the CMakeLists.txt appears to force 3.8, and the build is successful. 
```
find_package(Python3 3.8 EXACT COMPONENTS Interpreter Development)
```

**Comment by jonmaddock** [comment]
Hi @rmchurch, and thanks for reporting this issue. This might well be connected to #2832; can you check again once #2833 is merged?

**Comment by timothy-nunn** [comment]
Hi @rmchurch. To echo what Jon said, thank you for reporting this issue. I agree that this issue _should_ have been fixed by #2833, which I just merged. The new CMake policy looks for the Python version closest to the top of your `PATH`, which should always be the virtual environment version of Python. 

Further, I will correct the documentation as we do support Python 3.8 and 3.10, no longer just 3.8. I will check if Python 3.9 builds correctly as we should also support that.

Thanks again for the report, and do comment back here if the issue is not fixed on the latest version of main,
Tim

**Comment by timothy-nunn** [comment]
@rmchurch I am going to close this issue (as above, I believe the #2833 fixed it). Please get back in contact with any installation issues you experience in the future, and thank you again for bringing this issue to our attention.

Tim
